policy-test/src/lib.rs

```diff
@@ -100,6 +114,21 @@ pub async fn await_pod_ip(client: &kube::Client, ns: &str, name: &str) -> std::n
         .expect("pod IP must be valid")
 }
 
+#[tracing::instrument(skip_all, fields(%pod, %container))]
+pub async fn logs(client: &kube::Client, ns: &str, pod: &str, container: &str) {
+    let params = kube::api::LogParams {
+        container: Some(container.to_string()),
+        ..kube::api::LogParams::default()
+    };
+    let log = kube::Api::<k8s::Pod>::namespaced(client.clone(), ns)
+        .logs(pod, &params)
+        .await
+        .expect("must fetch logs");
+    for msg in log.lines() {
+        tracing::trace!(%msg);
+    }
```

Comment:
tiny nit: if we name the field `message` instead of `msg`, then `tracing-subscriber` will output it without a field name prefix:
```suggestion
    for message in log.lines() {
        tracing::trace!(%message);
    }
```